snippets/csharp/language-reference/keywords/switch/type-pattern.cs

```diff
@@ -28,28 +28,26 @@ private static void ShowCollectionInformation(object coll)
                break;
             case IEnumerable<int> ieInt:
                Console.WriteLine($"Average: {ieInt.Average(s => s)}");
-               break;   
+               break;
             case IList list:
                Console.WriteLine($"{list.Count} items");
                break;
             case IEnumerable ie:
                string result = "";
                foreach (var e in ie) 
-                  result += "${e} ";
+                  result += $"{e} ";
```

Comment:
This is the fix intended in the PR (in this file and the other two files), other changes are just whitespace changes.